.github/CONTRIBUTING.md

```diff
@@ -167,3 +167,71 @@ after the starting description.
  * ...
  */
 ```
+
+### Error Messages
+
+User-facing error messages should be clear, concise, and consistent. Error
+messages should be in sentence case but should not end with a period. Error
+messages should be free of grammatical errors and typos and written in American
+English.
+
+> ⚠️ Note that the error message style guide is a work in progress, and not all
+> the error messages have been updated to conform to the current styles. Error
+> message styles that should be followed:
+
+1. Messages should start with an upper case:
+
+```
+Bad: cannot parse input
+Good: Cannot parse input
+```
+
+2. Messages should not end with a period:
+
+```
+Bad: Cannot parse input.
+Good: Cannot parse input
+```
+
+3. Message should not use quotes for values:
```

Comment:
I wonder if this rule is reasonable. If the value is a string, using quotes around it doesn't seem like a problem to me.

How about not having this rule?

Reply:
IMHO I like the quotes too, so +1 from me. I just used what was suggested on the other PR and moved it here. I can update the PR if you'd like, however I'm at the lake with extremely bad internet this week, so I don't know if I'll be able to push the change. I'm back to civilization on Sunday, so I can do it then.

Reply:
I've updated the style guide to indicate that quotes should be used for values.